server/src/test/java/org/elasticsearch/action/admin/indices/mapping/get/GetMappingsResponseTests.java

```diff
@@ -83,22 +85,27 @@ protected GetMappingsResponse mutateInstance(GetMappingsResponse instance) throw
     public static ImmutableOpenMap<String, MappingMetaData> createMappingsForIndex() {
         // rarely have no types
         int typeCount = rarely() ? 0 : scaledRandomIntBetween(1, 3);
+        return createMappingsForIndex(typeCount);
```

Comment:
I'm not sure I understand when/why this would work? I can understand zero or one mappings for an index but shouldn't >1 be impossible/an error?